src/andAndDelete.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {
+  tasks, addTasks, createNewTask, repopulateList,
+} from './backEnd';
+import { taskList } from './index1';
+
+describe('It adds and deletes items from the list', () => {
+  const task1 = {
+    description: 'Car wash',
+    complemented: false,
+    index: 0,
+  };
+  const task2 = {
+    description: 'Laundary',
+    complemented: false,
+    index: 1,
+  };
+
+  const taskDescription = 'Pain the house';
+
+  const ul = document.createElement('ul');
+
+  test('it creates a list corresponding inputs', () => {
+    createNewTask(taskDescription);
+
+    expect(tasks[tasks.length - 1].description).toBe(taskDescription);
+
+    ul.appendChild(taskList(tasks[tasks.length - 1]));
+
+    expect(ul.innerHTML.includes(taskDescription)).toBe(true);
+  });
+
+  test('it adds and deletes an item from the To Do List in the DOM', () => {
+    const addedTask1 = addTasks(task1.description, task1.completed, task1.index);
+    const addedTask2 = addTasks(task2.description, task2.completed, task2.index);
+
```

Comment:
- [ ] Kindly add tests to check if it deletes more than one checked element in the DOM